src/sentry/tasks/assemble.py

```diff
@@ -295,22 +295,11 @@ def assemble_artifacts(org_id, version, checksum, chunks, **kwargs):
                 "dist": dist,
             }
 
-            saved_as_archive = False
-            if options.get("processing.save-release-archives"):
-                min_size = options.get("processing.release-archive-min-files")
-                if num_files >= min_size:
-                    try:
-                        update_artifact_index(release, dist, bundle)
-                        saved_as_archive = True
-                    except BaseException as exc:
-                        logger.error("Unable to update artifact index", exc_info=exc)
-
-            # NOTE(jjbayer): Single files are still stored to enable
-            # rolling back from release archives. Once release archives run
-            # smoothely, this call can be removed / only called when feature
-            # flag is off.
-            count_as_artifacts = not saved_as_archive
-            _store_single_files(archive, meta, count_as_artifacts)
+            min_size = options.get("processing.release-archive-min-files")
+            if num_files >= min_size:
+                update_artifact_index(release, dist, bundle)
```

Comment:
Not 100% sure if we can remove the catch-all here, but there haven't been any issues in the past few days apparently [1], so I guess it's fine.

[1] https://sentry.io/organizations/sentry/issues/?groupStatsPeriod=auto&query=%22Unable+to+update+artifact+index%22&statsPeriod=14d

Reply:
You have to, right? because you don't want to store single files anymore so you don't have a fallback path anyway

Reply:
ah I see, okay